coffee/base.py

```diff
@@ -833,3 +849,26 @@ def c_flat_for(code, parent):
     parent.children.append(FlatBlock(code))
     parent.children.append(new_block)
     return new_block
+
+
+# Access modes for a symbol ##
+
+
+class Access(object):
+
+    _modes = ["READ", "WRITE", "RW", "INC", "DEC", "IMUL", "IDIV"]
+
+    def __init__(self, mode):
+        self._mode = mode
+
+    def __eq__(self, other):
+        return self._mode == other._mode
+
+
+READ = Access("READ")
+WRITE = Access("WRITE")
+Rw = Access("RW")
```

Comment:
I just misspelled it, but there's already a commit fixing it on another branch (ast-if-nodes), which I hope will land soon